test/modules/elevationProfile/components/ElevationProfile.test.js

```diff
@@ -858,4 +858,24 @@ describe('ElevationProfile', () => {
 			expect(element._unsubscribers).toHaveSize(0);
 		});
 	});
+	it('removes highlghts', async () => {
+		// arrange
+		const coordinates = [
+			[0, 1],
+			[2, 3]
+		];
+
+		const element = await setup({
+			elevationProfile: {
+				active: true,
+				coordinates: coordinates
+			}
+		});
+
```

Comment:
Hmm, would suppose that  based on this test setup there can never be found highlight-features within the store